libraries/AP_Scripting/applets/Aerobatics/FixedWing/RateBased/sport_aerobatics.lua

```diff
@@ -29,13 +27,10 @@ local HGT_P = bind_add_param('HGT_P',1,1) -- height P gain
 local HGT_I = bind_add_param("HGT_I",2,1) -- height I gain
 local HGT_KE_BIAS = bind_add_param("KE_ADD",3,20) -- height knifeedge addition for pitch
 local THR_PIT_FF = bind_add_param("PIT_FF",4,80) -- throttle FF from pitch
-local SPD_P = bind_add_param("SPD_P",5,5) -- speed P gain
-local SPD_I = bind_add_param("SPD_I",6,25) -- speed I gain
```

Comment:
Should these be called but the result ignored so the parameters don't disappear? Or commented out so the numbers don't get reused?

Reply:
done.

Reply:
Added comments